### ISSUE TITLE
Redundant internet permission check

### ISSUE THREAD

**Issue by linxydmhg** [BracketCove/AndroidRestTutorial#1]
https://github.com/BracketCove/AndroidRestTutorial/blob/dd8ac26803d9add6fc134a58ddba77afa2439cf8/app/src/main/java/com/wiseassblog/androidresttutorial/main/MainActivity.java#L58

permission declared in manifest, right?

**Comment by MadCoderr** [comment]
Yes permission is declared in manifest, but because of api > 23 you need to ask permission in runtime from user

**Comment by linxydmhg** [comment]
Internet is not a dangerous permission that needs to be requested
